export_presets.cfg

```diff
@@ -0,0 +1,444 @@
+[preset.0]
+
+name="Android"
+platform="Android"
+runnable=true
+dedicated_server=false
+custom_features=""
+export_filter="all_resources"
+include_filter=""
+exclude_filter=""
+export_path="../Super Mario 63 Redux.apk"
+encryption_include_filters=""
+encryption_exclude_filters=""
+encrypt_pck=false
+encrypt_directory=false
+
+[preset.0.options]
+
+custom_template/debug=""
+custom_template/release=""
+gradle_build/use_gradle_build=false
+gradle_build/export_format=0
+gradle_build/min_sdk=""
+gradle_build/target_sdk=""
+architectures/armeabi-v7a=true
+architectures/arm64-v8a=true
+architectures/x86=false
+architectures/x86_64=false
+version/code=7
+version/name="0.1.6"
+package/unique_name="com.sm63.redux"
+package/name="Super Mario 63 Redux"
+package/signed=true
+package/app_category=2
+package/retain_data_on_uninstall=true
```

Comment:
Out of curiosity, what data is there to retain on uninstall? Is it just settings, or is this looking ahead to user-created levels?

Reply:
Settings, and yes, user levels.